skyportal/models/analysis.py

```diff
@@ -142,6 +170,13 @@ class AnalysisService(Base):
         ),
     )
 
+    obj_analyses = relationship(
+        'ObjAnalysis',
+        back_populates='analysis_service',
+        passive_deletes=True,
+        doc="The Obj Analyses associated with this Service.",
```

Comment:
Maybe something like `Instances of analysis applied to specific objects`?

Reply:
Done.